### DISCUSSION TITLE
Deploy settings.json for all users

### DISCUSSION THREAD

**Discussion by Tchad90** [gildas-lormeau/SingleFile#1046] [General]
Hello,

I have hundreds of users of the singlefile addon for my client. He provided me his json configuration file. The goal is to deploy the configuration for each of these users (via group policy)

I searched in appdata/local/Google/chrome/Default/extension but I don't see a way to do this other than graphically.

Can you help me please?

PS: Currently the addon is deployed on Google Chrome

**Comment by gildas-lormeau** [comment]
Looking at this documentation https://support.google.com/chrome/a/answer/9867568?hl=en&ref_topic=9023098, it looks like it's technically impossible to pass some data to an extension installed via a group policy. It looks like the only option is to fork SingleFile, apply the changes in the source code, and publish the fork on the store.

**Comment by gildas-lormeau** [comment]
I'm moving this issue in the "Discussions" tab
